test/system_test.py

```diff
@@ -625,21 +606,27 @@ def test_install_version_1_0_1_1(self):
     def test_install_version_1_0_1_1b(self):
         spack_install_and_test('py-gt4py @1.0.1.1b')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_1_6(self):
         spack_install_and_test('py-gt4py @1.0.1.6')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_1_7(self):
         spack_install_and_test('py-gt4py @1.0.1.7')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_3(self):
         spack_install_and_test('py-gt4py @1.0.3')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_3_1(self):
         spack_install_and_test('py-gt4py @1.0.3.1')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_3_2(self):
         spack_install_and_test('py-gt4py @1.0.3.2')
 
+    @pytest.mark.no_balfrin  # problem with gt4py and spack v21.1
     def test_install_version_1_0_3_3(self):
         spack_install_and_test('py-gt4py @1.0.3.3')
```

Comment:
This means, these tests are run on **no** machine. Therefore this is dead code. Please remove it.

Reply:
not dead anymore ;-)